drivers/iio/adc/tswizard_adc.c

```diff
@@ -149,21 +149,21 @@ static int ts_supervisor_adc_probe(struct platform_device *pdev)
 	return devm_iio_device_register(&pdev->dev, indio_dev);
 }
 
-static const struct of_device_id tssupervisor_of_match[] = {
-	{ .compatible = "technologic,tssupervisor-adc", },
+static const struct of_device_id tswizard_of_match[] = {
+	{ .compatible = "technologic,tswizard-adc", },
```

Comment:
This seems to be a potential issue. I noted in my previous comment that `technologic,wizard-adc` was used, but this looks like its matching `technologic,tswizard-adc`. This should be rectified. But is also strange because this change goes from `technologic,tssupervisor-adc` but the change above in the devicetree changed from `technologic,supervisor-adc`.